### DISCUSSION TITLE
Does the extractor support F#?

### DISCUSSION THREAD

**Discussion by nkosi23** [perpetualKid/GetText.NET#51] [Q&A]
Hello,

Your library looks very interesting! Since my project is developed using F# I am wondering if GetText.NET would be able to extract translatable strings from F# (.fs) source files?

**Comment by perpetualKid** [comment]
The extractor is based on C# code analyzer and syntax tree parser, which I don't think will support any F# language syntax.

I don't know if there is an F# equivalent, documentation only shows C# and VB syntrax tree parsers 
https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.syntaxtree?view=roslyn-dotnet-4.6.0
